nncf/quantization/algorithms/fast_bias_correction/torch_backend.py

```diff
@@ -0,0 +1,142 @@
+    }
+
+    @property
+    def operation_metatypes(self) -> Registry:
+        return PT_OPERATOR_METATYPES
+
+    @property
+    def tensor_processor(self) -> PTNNCFCollectorTensorProcessor:
+        return PTNNCFCollectorTensorProcessor()
+
+    @staticmethod
+    def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> PTTargetPoint:
+        if NNCFGraphNodeType.INPUT_NODE in target_node_name or target_type == TargetType.POST_LAYER_OPERATION:
+            port_id = None
+        if target_type in PTFastBiasCorrectionAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP:
+            target_type = PTFastBiasCorrectionAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP[target_type]
+        return PTTargetPoint(target_type, target_node_name, input_port_id=port_id)
+
+    @staticmethod
+    def create_bias_correction_command(
+        node: NNCFNode, bias_value: np.ndarray, nncf_graph: NNCFGraph
+    ) -> PTBiasCorrectionCommand:
+        return create_bias_correction_command(node, bias_value)
+
+    @staticmethod
+    def model_extraction_command(inputs: List[str], outputs: List[str]) -> PTModelExtractionWithFusedBiasCommand:
+        return PTModelExtractionWithFusedBiasCommand(inputs[0])
+
+    @staticmethod
+    def mean_statistic_collector(
+        reduction_shape: ReductionShape,
+        inplace: bool,
+        num_samples: Optional[int] = None,
+        window_size: Optional[int] = None,
+    ) -> PTMeanStatisticCollector:
+        return PTMeanStatisticCollector(reduction_shape, num_samples, window_size)
+
+    @staticmethod
+    def get_sub_input_output_names(subgraph: NNCFNetwork) -> Tuple[str, str]:
+        # Pytorch does not have name for extracted node
+        return None, None
+
+    @staticmethod
+    def create_input_data(
+        shape: Tuple[int], data: List[torch.Tensor], input_name: str, channel_axis: int
+    ) -> torch.Tensor:
+        blob = torch.zeros(shape, dtype=data[0].dtype)
+        for j, idx in enumerate(np.ndindex(blob.shape[channel_axis])):
+            index = tuple(slice(None) if i != channel_axis else idx for i in range(blob.ndim))
+            blob[index] = data[j]
+        return blob
+
+    @staticmethod
+    def get_bias_value(node: NNCFNode, nncf_graph: NNCFGraph, model: NNCFNetwork) -> np.ndarray:
+        return get_fused_bias_value(node, model)
+
+    @staticmethod
+    def get_activation_port_ids_for_bias_node(node: NNCFNode) -> Tuple[int, int]:
+        return 0, 0
+
+    @staticmethod
+    def process_model_output(raw_data: Dict, output_name: str) -> PTNNCFTensor:
+        return PTNNCFTensor(raw_data)
+
+    @staticmethod
+    def is_quantized_weights(node: NNCFNode, nncf_graph: NNCFGraph, model: NNCFNetwork) -> bool:
+        return is_quantized_weights(node, model)
+
+    @staticmethod
+    def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph, model: NNCFNetwork) -> bool:
+        return is_node_with_fused_bias(node, model)
```

Comment:
Could you briefly explain why you cannot determine fused bias by NNC Graph? I guess such functionality can be general and implemented during collecting layer attributes.

Reply:
I'm not sure that it's need for general implementation, it looks like torch specific case.

Reply:
We are going to minimize backend interfaces but this PR extending it. One way is used backed agnostic abstractions such as NNCFGraph, to do it. Thus I want to understand why we cannot extend NNCFGraph and implement algorithm that will find bias nodes based NNCFGraph, because It looks like the general functionality between frameworks. Let's discuss it offline, may be we will find solution.